gio/src/auto/debug_controller_dbus.rs

```diff
@@ -0,0 +1,114 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use crate::Cancellable;
+use crate::DBusConnection;
+use crate::DBusMethodInvocation;
+use crate::DebugController;
+use crate::Initable;
+use glib::object::Cast;
+use glib::object::IsA;
+use glib::signal::connect_raw;
+use glib::signal::SignalHandlerId;
+use glib::translate::*;
+use glib::StaticType;
+use std::boxed::Box as Box_;
+use std::fmt;
+use std::mem::transmute;
+use std::ptr;
+
+glib::wrapper! {
+    #[doc(alias = "GDebugControllerDBus")]
+    pub struct DebugControllerDBus(Object<ffi::GDebugControllerDBus, ffi::GDebugControllerDBusClass>) @implements DebugController, Initable;
+
+    match fn {
+        type_ => || ffi::g_debug_controller_dbus_get_type(),
+    }
+}
+
+impl DebugControllerDBus {
+    pub const NONE: Option<&'static DebugControllerDBus> = None;
+
+    #[doc(alias = "g_debug_controller_dbus_new")]
+    pub fn new(
+        connection: &DBusConnection,
+        cancellable: Option<&impl IsA<Cancellable>>,
+    ) -> Result<Option<DebugControllerDBus>, glib::Error> {
+        unsafe {
+            let mut error = ptr::null_mut();
+            let ret = ffi::g_debug_controller_dbus_new(
+                connection.to_glib_none().0,
+                cancellable.map(|p| p.as_ref()).to_glib_none().0,
+                &mut error,
+            );
+            if error.is_null() {
+                Ok(from_glib_full(ret))
+            } else {
+                Err(from_glib_full(error))
+            }
+        }
+    }
+}
+
+pub trait DebugControllerDBusExt: 'static {
+    #[cfg(any(feature = "v2_72", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v2_72")))]
+    fn connection(&self) -> Option<DBusConnection>;
```

Comment:
Probably not nullable

Reply:
Implemented it manually

Reply:
You can set `nullable = false` in `Gir.toml` instead